src/components/Header/typescript/sidebar.constants.ts

```diff
@@ -0,0 +1 @@
+export const BENCHTTP = 'Benchttp'
```

Comment:
Maybe put this file in `shared` folder, if we need for the title of website for example